.github/workflows/lint.yml

```diff
@@ -68,3 +68,14 @@ jobs:
         )"
         echo "$issues"
         [ "$issues" == "" ] || exit 1
+
+  ensure-sha-pinned-actions:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+      - name: Ensure SHA pinned actions
+        uses: zgosalvez/github-actions-ensure-sha-pinned-actions@4830be28ce81da52ec70d65c552a7403821d98d4 # v3
+        with:
+          allowlist: |
+            Open-MSS/
+          dry_run: 'false'
```

Comment:
```suggestion
```

Reply:
Also for true it was with 'true' correct interpreted, but that would been expected behaviour. 

https://github.com/Open-MSS/MSS/pull/2802/commits/0532a4a2a9ecddb2d25220b3469f7094039e24e9